src/modules/sequencer-crosshair/sequencer-crosshair.js

```diff
@@ -0,0 +1,58 @@
+import Crosshairs from "./Crosshairs.js";
+
+export default class Crosshair {
+
+	/**
+	 * @param {CrosshairData/Object} config
+	 * @param {CrosshairCallbackData/Object} callbacks
+	 * @returns {*}
+	 */
+	static show(config = {}, callbacks = {}) {
+
+		const reticle = new Crosshairs({
+			t: CONST.MEASURED_TEMPLATE_TYPES.CIRCLE,
+			distance: config?.distance ?? 0.5,
+			borderColor: config.borderColor ?? "#000000",
+			fillColor: config.fillColor ?? "#000000",
+		}, {
+			parent: canvas.scene
+		}, foundry.utils.mergeObject(Crosshairs.defaultConfig(), config), callbacks);
+
+		return reticle.show();
+
+	}
+
+	/**
+	 * @param {CrosshairData} crosshair
+	 * @param {String/Array<string>} types
+	 * @param {Function} filterMethod
+	 * @returns {Array<Document>|Object<String:Array<Document>>}

```

Comment:
I'm not sure how accurate you care the types to be but they'd be
```suggestion
	 * @param {CrosshairData} crosshair
	 * @param {String|Array<string>} types
	 * @param {(object: PlaceableObject, crosshair: CrosshairData) => boolean} filterMethod
	 * @returns {Array<Document>|Record<String, Array<Document>>}

```